apps/aehttp/test/aehttp_spec_SUITE.erl

```diff
@@ -73,3 +75,31 @@ get_api(Config) ->
     {ok,{{"HTTP/1.1",405,"Method Not Allowed"},_,_}} =
          httpc:request(post, {URL, [], Type, Body}, [], []),
     ok.
+
+validate_api(Config) ->
+    Spec = json_from_yaml(Config),
+
+    Url = "http://validator.swagger.io/validator/",
+    {ok, {_, _Headers, Body}} = httpc:request(post, {Url, [],  "application/json", Spec}, [], []),
+    %% For manual verification visit https://github.com/swagger-api/validator-badge
+    %% A picture is returned in the body
+    case aec_hash:blake2b_256_hash(list_to_binary(Body)) of
```

Comment:
Compairing against a picture? Would it be an alternative to use the `debug/` endpoint instead and not have to rely on that badge picture?